pyproject.toml

```diff
@@ -24,6 +24,12 @@ classifiers = [
 dynamic = [
     "version",
 ]
+dependencies = [
+    "json5",
+    "tabulate",
+    "colorama; os_name == 'nt'",
```

Comment:
Interesting. I see  `os_name` and `platform_python_implementation` are part of the [grammer](https://peps.python.org/pep-0508/#grammar), but I am used to seeing them spelled as `sys_platform` (which would be `win32`) and `implementation_name` (which would be `!= "pypy").

Reply:
I'm most used to the `sys_platform` one, but in the `asv` codebase there's a variable `WIN = os.name` so I thought I'd go with this here 😅